css-layout-classes/styles.css

```diff
@@ -0,0 +1,44 @@
+.container {
+  max-width: 800px;
+  margin: auto;
+
+  /* margin-right: auto;
+  margin-left: auto; */
+}
+
+.row {
+  display: flex;
+}
+
+.coulmn-third-first {
+  /* flex-basis: calc (100% / 4);
+  flex-wrap: wrap; */
+}
+
+.column-second {
+  width: calc(100% / 3);
+}
```

Comment:
I'm not sure what `column-second` is meant to mean, column classes are supposed to be named after their size.